latexrun

```diff
@@ -221,6 +222,14 @@ def verbose_cmd(args, cwd=None, env=None):
         print(cmd, file=sys.stderr)
 verbose_cmd.enabled = False
 
+def mkdir_p(path):
+    try:
+        os.makedirs(path)
+    except OSError as exc: # Python >2.5
```

Comment:
Please either omit the comment (there are plenty of other things that require 2.6 in this file) or, if you really want to keep it, write "Python >= 2.6" since there were plenty of Python 2.5 point releases that are >2.5, but aren't 2.6.